doc/dev/radosgw/s3_objects_dedup.rst

```diff
@@ -0,0 +1,95 @@
+======================
```

Comment:
this is mostly a "user doc" (which is great). please move it here: https://github.com/ceph/ceph/tree/main/doc/radosgw

* please change the "memory usage" section to contain onlyobject count and memory (and remove the explanations).
* please add information about the stats output that will help the user understand what they mean (when they are not trivial)
* please add more info on skipped object
* add the design doc here: ` doc/dev/radosgw`

Reply:
I moved the document with some changes, will keep editing tomorrow